scrape/scrape.go

```diff
@@ -132,6 +132,20 @@ var (
 			Help: "How many times a scrape cache was flushed due to getting big while scrapes are failing.",
 		},
 	)
+	targetMetadataCacheBytes = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "prometheus_target_metadata_cache_bytes",
+			Help: "The number of bytes that are currently used for storing metric metadata of a target",
```

Comment:
Sorry, not sure I follow your comment. Yes, the target information is represented as labels - are we saying that the usage of the word "target" in this context does not make sense because of how it is represented in-code? 

Would the same sentence without the last three words work?

Reply:
Reworded, what do you think now?